learn/app-development/widgets/datalive/form/toggle.md

```diff
@@ -5,17 +5,16 @@ id: ""
 
```

Comment:
Missing sidebar

Reply:
These are all duplicates, so removed them from the repo.